src/Cell.cpp

```diff
@@ -764,6 +615,16 @@ void Cell::setName(const char* name) {
 
 
 /**
+ * @brief Sets the name of the Cell
+ * @param name the Cell name string
+ */
+void Cell::setRegion(Region* region) {
+  // FIXME: this is a memory leak
+  _region = region;
```

Comment:
An alternative option might be to clone any `Region` assigned to a `Cell` and leave the Python garbage collector to clean up any `Region`s instantiated in Python.